base/src/main/java/io/spine/io/Resource.java

```diff
@@ -83,8 +132,7 @@ public boolean exists() {
      * @return the resource URL
      */
     public URL locate() {
-        URL resource = classLoader().getResource(path);
-        checkFound(resource);
+        URL resource = toUrl();
```

Comment:
Please inline `toUrl()` into `locate()`. There is no point in having two methods.